On preempt-rt there is a new file-system API where
nrpages has been replace by an atomic __nrpages with
a corresponding initialisation routine.

Fix for preempt RT.

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 fs/ubifs/dir.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.24/fs/ubifs/dir.c
===================================================================
--- linux-2.6.24.orig/fs/ubifs/dir.c	2009-01-26 18:49:52.000000000 +0100
+++ linux-2.6.24/fs/ubifs/dir.c	2009-01-26 19:08:26.000000000 +0100
@@ -114,7 +114,7 @@ struct inode *ubifs_new_inode(struct ubi
 	inode->i_mode = mode;
 	inode->i_mtime = inode->i_atime = inode->i_ctime =
 			 ubifs_current_time(inode);
-	inode->i_mapping->nrpages = 0;
+	mapping_nrpages_init(inode->i_mapping);
 	/* Disable readahead */
 	inode->i_mapping->backing_dev_info = &c->bdi;
 
